index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+ 
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+
+    <header>
+        <nav>
+
+            <a href="#"><h2 id="welcome">Welcome</h2></a>
+            <div class="nav-links">
+            <i id="menu" class="material-symbols-outlined">menu</i>
+            </div>
+        </nav>
+    </header>
+        <section class="headline">
+            <h1 class="black">Hey there. I’m Jemutai.<br><span id="green">I’m a software developer</span>
+                </h1>
+                <p class="text"> 
+                    I can help you build a product , feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br>
+                   don’t hestiate to contact me.
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#"><img src="images/Vector.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Linkedinicon.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector1.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector2.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector3.png" alt="social media icons"></a>
+                    </li>
+                </ul>
+
+        </section>
+    
+</body>
```

Comment:
- [OPTIONAL ] I think it would be better if you don't leave empty space within you file. Leaving empty spaces in your files would defeat HTML semantics. If you need more space between your element, you can increase line height in your vscode text editor or any text editor you chose. You can follow this [link](https://www.youtube.com/watch?v=PD_aNN29F44) to resolve that.

Reply:
- [ ] Kindly compare the Figma design with the outlook of your project at the moment and try to align your project with the provided Figma design. You can do it I trust you :mechanical_arm: 

Figma Design

![figma-temp-1](https://user-images.githubusercontent.com/26597432/185364497-de4ed217-78cb-4a8a-a10a-907cccb78a7e.png)

Your Design

![your-design-1](https://user-images.githubusercontent.com/26597432/185364823-4a27df48-b241-46f1-bb5a-93249514cb30.png)


- [ ] There is also an horizontal scroll bar at the bottom of your page. Kindly fix that too before you ask for another review.